### DISCUSSION TITLE
Resolving "Server returned 401 Unauthorized (authorization failed)" for IP camera

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5351] [General]
When an IP camera RTSP URL is added to AMS as a stream source but the broadcast doesn't start and the logs entries show

`rtsp://usernmae:password@1.2.3.4:554/Streaming/Channels/102 with error:: Server returned 401 Unauthorized (authorization failed)`
Even if tried with `ffmpeg` or `ffplay` it still gives same error

It is mainly due to the RTSP Authentication type settings on the camera itself and changing it to Basic resolves the issue.
![image001](https://github.com/ant-media/Ant-Media-Server/assets/86982446/a46df019-24da-41b8-b4d7-209f282ef4cc)

Cheers!

**Comment by Pi-Boss** [comment]
I see this problem, too  
But the camera doesn't have that rtsp digest algorihm。emm，I don't know how to solve it

**Reply by Mohit-3196** [reply]
Hi @Pi-Boss,
What authentication type does your camera have?

Thank you

**Comment by BenBuaronMS** [comment]
Same issue here.

**Reply by Mohit-3196** [reply]
Hi @BenBuaronMS,
How are you doing?

Can you please tell me what issue you see with your camera and what kind of authentication it has?
Also, is it possible to access the camera?

Thank you.

**Comment by CyberKuzya** [comment]
maybe it will be useful to someone.
To receive rtsp stream from camera in VLC it was necessary to set the setting to specify the type of stream RTSP(TCP), however in zoneminder via tcp it is impossible to receive stream, in logs error 401. Solution - change in zoneminder camera protocol to UDP.

**Reply by avu3** [reply]
I wanted to thank you for this.  It is likely a firmware issue on the camera - but I have one (out of a number of the same model) that refuses to work unless I set it to UDP.  The rest work fine on TCP.  Out of desperation I tried this, and it worked!
Like others, the same format URL works for other cameras.  The URL for this camera works in a live preview in VLC.
I also tried changing the security settings on the Camera as suggested elsewhere on other forms. Nothing worked except UDP. So, thank you so much for sharing.
